week-8/Homework/mandatory/2-fetch-exercise/exercise.js

```diff
@@ -17,10 +17,12 @@ Open index.html in your browser. Every time you refresh the page,
 a different greeting should be displayed in the box.
 */
 
-fetch('*** Write the API address here ***')
-    .then(function(response) {
+fetch('https://codeyourfuture.herokuapp.com/api/greetings')
+    .then(response => {
         return response.text();
     })
-    .then(function(greeting) {
-        // Write the code to display the greeting text here
-    });
\ No newline at end of file
+    .then(greeting => {
+        console.log(greeting);
+        let randomGreeting = document.getElementById("greeting-text").innerText = greeting;
+        return randomGreeting;
```

Comment:
This works well, good job :)

Can you talk through what this function will return?